tests/drivers/can/timing/src/main.c

```diff
@@ -186,6 +183,8 @@ static void test_timing_values(const struct device *dev, const struct can_timing
 	if (test->invalid) {
 		zassert_equal(sp_err, -EINVAL, "err %d, expected -EINVAL", sp_err);
 		printk("OK\n");
+	} else if (sp_err == -ENOTSUP) {
+		printk("bitrate not supported\n");
```

Comment:
Yeah. This is unlikely, though, as the CAN controller would have to not support _any_ of the bitrates listed in the table (could be a programming error, of course).

I am planning further improvements to the timing tests as I'd like them to always use the default sample points and cover all CiA recommended bitrates. This will include splitting what is currently one test case iterating through the list of bitrates to a number of indidual tests, each covering a given bitrate. That should also make it easier to ensure that at least one of them pass.

Reply:
I can leave out the change to the timing tests from this PR, if you think it needs more work.

Reply:
Sounds good, no I woudn't leave it out then.